src/Chainweb/Pact/Types.hs

```diff
@@ -362,7 +361,6 @@ data PactServiceEnv tbl = PactServiceEnv
     -- ^ The limit of checkpointer's rewind in the `execValidationBlock` command.
     , _psOnFatalError :: forall a. PactException -> Text -> IO a
     , _psVersion :: ChainwebVersion
-    , _psValidateHashesOnReplay :: !Bool
```

Comment:
I propose to keep this with `True` as default. It can then be disabled locally where it is safe and beneficial.